source/_integrations/amcrest.markdown

```diff
@@ -93,7 +93,7 @@ authentication:
   type: string
   default: basic
 scan_interval:
-  description: Defines the update interval of the sensor in seconds.
+  description: Defines the update interval of the sensor in seconds. This setting is fixed from HA version 108.5 and scan_interval has no effect
```

Comment:
This is a bit weird? According to the code it is still there?

If it has no effect, a bug should be raised.

Reply:
Hi Frenck,

Please see comment by @pnbruckner
https://github.com/home-assistant/core/issues/33875#issuecomment-617189318

But it seems although this was closed "wontfix"
https://github.com/tchellomello/python-amcrest/issues/155

That @pnbrucker has had a change of heart
https://github.com/home-assistant/core/pull/36955

So this markdown needs dropping.

James